### PR TITLE
Ability to change route name using `pageTitle`, `navTitle`

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
- Requires [skyux-builder-plugin-stache-route-metadata](https://github.com/blackbaud/skyux-builder-plugin-stache-route-metadata).
- Need to think about how we can also manage route order using this method (in a different PR)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,11 +33,12 @@
     "@blackbaud/skyux": "2.0.0-beta.27",
     "@blackbaud/skyux-builder-plugin-stache-code-block": "1.0.0",
     "@blackbaud/skyux-builder-plugin-stache-include": "1.0.0",
+    "@blackbaud/skyux-builder-plugin-stache-route-metadata": "blackbaud/skyux-builder-plugin-stache-route-metadata#unit-tests",
```
Comment (Blackbaud-BrandonJones) [inline package.json]: are we pointing to #unit-tests on purpose?

File: package.json [hunk 2]
```diff
@@ -33,11 +33,12 @@
...
     "prismjs": "1.6.0",
     "smoothscroll-polyfill": "0.3.5"
   },
   "devDependencies": {
-    "@blackbaud/skyux-builder": "1.0.0-beta.25",
+    "@blackbaud/skyux-builder": "blackbaud/skyux-builder#plugin-config",
```
Comment (Blackbaud-BrandonJones) [inline package.json]: also plugin-config?

File: package.json (hunk 1 above)
Comment (Blackbaud-SteveBrush) [inline package.json]: Yeah, until y'all approve that PR, too :)

File: package.json (hunk 2 above)
Comment (Blackbaud-SteveBrush) [inline package.json]: This can be updated. One minute...

File: src/app/public/src/modules/shared/route-metadata.service.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// The contents of this file are automatically generated by
```
Comment (blackbaud-brandonhare) [inline src/app/public/src/modules/shared/route-metadata.service.ts]: Good use of comments!

**Comment by Blackbaud-SteveBrush** [comment]
https://github.com/blackbaud/stache2/issues/168

**Pull request merged**